### PR TITLE
Add a Governance section

### PR DESCRIPTION
**Pull request by azeey**
## Summary

Adds a governance section inspired by https://docs.ros.org/en/jazzy/The-ROS2-Project/Governance.html that describes what the various governance structures (OSRA, TGC, PMC, Committers) and starts the list of PMC members and Committers.

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: governance.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+Technical Governance Committee provides oversight of the project, while the
+Project Management Committee is responsible for day-to-day operations. This
+document outlines these entities and their respective roles and
+responsibilities. For further information, please refer to
+[OSRA website](https://osralliance.org/how-it-works/).
+
+## Open Source Robotics Alliance (OSRA)
+
+The Open Source Robotics Foundation (OSRF) established the Open Source Robotics
+Association (OSRA) to enhance the governance of its open-source projects
```
Comment (caguero) [inline governance.md:14]:
```suggestion
Alliance (OSRA) to enhance the governance of its open-source projects
```

**Review by caguero — COMMENTED** [review]
Minor suggestion, otherwise looks great.

**Review by iche033 — COMMENTED** [review]
looks good to me

File: governance.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+The Gazebo PMC currently consists of the following individuals:
+
+| Name              | Affiliation                            | Github Handle                     | PMC Role       | Time Zone (optional)   |
+| ----------------- | -------------------------------------- | --------------------------------- | -------------- | ---------------------- |
+| Addisu Z. Taddese | [Intrinsic](https://www.intrinsic.ai/) | [azeey](https://github.com/azeey) | Project Leader | CST (UTC-6)/CDT(UTC-5) |
+
+## Current Gazebo Committers
+
+The Gazebo Committers (who are not also PMC members) currently consists of the
```
Comment (scpeters) [inline governance.md:57]:
```suggestion
The Gazebo Committers (who are not also PMC members) currently consist of the
```

**Review by bperseghetti — APPROVED** [review]

**Pull request merged**